### ISSUE TITLE
Update public site

### ISSUE THREAD

**Issue by cquiroz** [typelevel/squants#159]
The site at `www.squants.com` still references the old organization and version. The `gh-pages` branch contains raw html for the pages instead of a jekyll site

What do you think about migrating to `sbt-microsites` I'd really like the documentation to be typechecked with `tut`. This is also suggested by typelevel at

https://github.com/typelevel/general/blob/master/governance/membership.md

I'm not sure how to keep the nice layout of the current site with `sbt-microsites` though

**Comment by derekmorr** [comment]
We also need to update the site to reflect the current version, 1.0.

**Comment by cquiroz** [comment]
Yes, that was the original motivation for this issue. Not only the version but also the new organization

**Comment by garyKeorkunian** [comment]
I used the github.io pages feature to publish this originally, but I believe that is limited to a single site per account.  It does seem to allow multiple project pages.  It does not seem I am able to republish the readme from the typelevel/squants repo.

The currently typelevel.github.io redirects to typelevel.org.

Wherever we do eventually host the page, I am happy to maintain the squants.com/squants.org domains and point them there.  In the meantime, I am going to update the DNS to redirect those to GitHub repo.
